src/pages/contracts/_components/VerifiedContractRow.tsx

```diff
@@ -0,0 +1,139 @@
+import LinkText from "@components/commons/LinkText";
+import { VerifiedContract } from "mockdata/VerifiedContractData";
+import { truncateTextFromMiddle } from "shared/textHelper";
+
+export default function VerifiedContractRow({
+  data,
+}: {
+  data: VerifiedContract;
+}) {
+  return (
+    <div>
```

Comment:
can we remove this `<div>`?

Reply:
mot able to remove this div as the return statement requires the div (wrapper) to be returned

Reply:
okay, then let's use react fragment `<>`

Reply:
After the new change, can we change this to return fragment?